db/schema.rb

```diff
@@ -90,17 +91,8 @@
     t.integer "time_limit", default: 60
   end
 
-  create_table "users", force: :cascade do |t|
-    t.string "name"
-    t.integer "unit"
-    t.string "email"
-    t.string "phone"
-    t.boolean "active"
-    t.boolean "admin"
-    t.datetime "created_at", precision: 6, null: false
-    t.datetime "updated_at", precision: 6, null: false
-    t.boolean "parking_admin"
-  end
+# Could not dump table "users" because of following StandardError
```

Comment:
This seems like an error with an invalid column type?

Reply:
Might it have something to do with the difference in owner and user type?

Reply:
This file should be deleted as we switched to `structure.db`